static-instrumentation/settings.gradle

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Remove this file, let's keep all th eincludes at the top level

Reply:
🆗